ci-operator/step-registry/redhat-chaos/pod-scenarios/ovn-cp-disruption/redhat-chaos-pod-scenarios-ovn-cp-disruption-ref.yaml

```diff
@@ -0,0 +1,69 @@
+ref:
+  as: redhat-chaos-pod-scenarios-ovn-cp-disruption
+  from_image:
+    namespace: chaos
+    name: prow-scripts
+    tag: latest
+  commands: redhat-chaos-pod-scenarios-ovn-cp-disruption-commands.sh
+  credentials:
+  - namespace: test-credentials
+    name: cluster-secrets-aws-chaos
+    mount_path: /secret/telemetry
+  - namespace: test-credentials
+    name: ocp-qe-perfscale-es
+    mount_path: /secret/es
+  env:
+  - name: TARGET_NAMESPACE
+    default: "openshift-ovn-kubernetes"
+  - name: POD_LABEL
+    default: "app=ovnkube-control-plane"
+  - name: DISRUPTION_COUNT
+    default: "1"
+  - name: EXPECTED_POD_COUNT
+    default: "3"
+  - name: KILL_TIMEOUT
+    default: "180"
+  - name: EXPECTED_RECOVERY_TIME
```

Comment:
@shahsahil264 is it captured from at least 10 manual runs? Thoughts?

Reply:
Hey @chaitanyaenr ,  not yet, I'll do the manual runs and change it!